cpp/harvest/nacjd.cc

```diff
@@ -119,6 +135,17 @@ void HandleChar(const char c) {
     }
 }
 
+std::string CheckingStatisticsCategory(std::string data_title) {
```

Comment:
data_title should be const

Reply:
And a better name would be either CheckStatisticsCategory or GetStatisticsCategory